src/core/Akka.Persistence/Persistence.cs

```diff
@@ -252,6 +253,23 @@ public IActorRef JournalFor(string journalPluginId)
             return PluginHolderFor(configPath, JournalFallbackConfigPath).Ref;
         }
 
+        /// <summary>
+        /// Shortcut for invoking journal health checks.
+        /// </summary>
+        /// <param name="journalPluginId">The HOCON id of the Akka.Persistence plugin./</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>A <see cref="PersistenceHealthCheckResult"/> with health status and possibly a descriptive message.</returns>
+        public async Task<PersistenceHealthCheckResult> CheckJournalHealthAsync(string journalPluginId,
```

Comment:
Convenience method for invoking the health check - if the `journalPluginId` is not found this method will err.